packages/api/src/useSubscription.ts

```diff
@@ -43,23 +46,29 @@ const useSubscription = <T extends TSocketSubscribableEndpointNames>(name: T, id
                         setIsLoading(false);
                     }
                 );
+                subscribers.current.set(subscriber, subscriber.unsubscribe);
+
+                return () => {
+                    subscriber.unsubscribe?.();
+                    subscribers.current.delete(subscriber);
+                };
```

Comment:
If we make separate function called unsubscribe that receive id will it be difficult to implement? 🤔 . So the subscribe return subscription_id from BE `response.subscription.id`

Reply:
The problem is that we don't receive `subscription_id` right away, but in the first `onData`.

<img width="1117" alt="Screenshot 2023-12-19 at 12 20 32" src="https://github.com/binary-com/deriv-app/assets/119863957/5de14d49-6c41-4cbc-8d3d-32df67e604ee">

<img width="534" alt="Screenshot 2023-12-19 at 12 13 17" src="https://github.com/binary-com/deriv-app/assets/119863957/b164bf12-9d93-4b94-ac6c-1d14f0b916cf">

We don't get an identifier upon subscription and also I don't see a point of keeping it, because the only handle we need is the `unsubscribe` callback imo